commit bdc775fdd8acdad5c58ff3065a21396f80ce5db4
Author: John Spray <jspray@redhat.com>
Date:   Mon Jul 31 09:24:09 2017 -0400

    mgr: apply a threshold to perf counter prios
    
    ...so that we can control the level of load
    we're putting on ceph-mgr with perf counters.  Don't collect
    anything below PRIO_USEFUL by default.
    
    Signed-off-by: John Spray <john.spray@redhat.com>

diff --git a/src/mgr/DaemonServer.cc b/src/mgr/DaemonServer.cc
index ff520b17359..3972946d8c0 100644
--- a/src/mgr/DaemonServer.cc
+++ b/src/mgr/DaemonServer.cc
@@ -318,6 +318,7 @@ bool DaemonServer::handle_open(MMgrOpen *m)
 
   auto configure = new MMgrConfigure();
   configure->stats_period = g_conf->mgr_stats_period;
+  configure->stats_threshold = g_conf->get_val<int64_t>("mgr_stats_threshold");
   m->get_connection()->send_message(configure);
 
   DaemonStatePtr daemon;
